zerver/lib/test_helpers.py

```diff
@@ -38,9 +38,9 @@
     UserProfile,
 )
 
-if False:
+if TYPE_CHECKING:
     # Avoid an import cycle; we only need these for type annotations.
-    from zerver.lib.test_classes import ZulipTestCase, MigrationsTestCase
+    from zerver.lib.test_classes import ZulipTestCase, MigrationsTestCase  # nocoverage
```

Comment:
We should just add TYPE_CHECKING to the set of things that are always excluded from coverage in `coveragerc`; I'll do that and merge this one.